app/models/user.rb

```diff
@@ -32,14 +40,4 @@ def full_name
     return username unless first_name.present?
     "#{first_name} #{last_name}"
   end
-
-  protected
-
-  def password_required?
-    !password.nil? || !password_confirmation.nil?
```

Comment:
You have to generate a random password in https://github.com/toptierlabs/rails_api_base/blob/master/app/models/concerns/facebookeable.rb#L11, because facebook users don't have passwords